src/components/icon/icon.tsx

```diff
@@ -65,15 +65,10 @@ export class Icon {
      * will remain the way they look in the file.
      * Therefore we inject the svg as inline markup instead.
      */
-    private renderSvg(svgData: string, name: string) {
+    private renderSvg(svgData: string) {
```

Comment:
What the heck? Building this locally (with `npm run build`) fails, because I forgot to remove the `name` argument in the call. But it passes `npm run build` in the CI job in GitHub Actions?! That's not good…

Reply:
(Now I've pushed the fix, so it should be green. But [this run](https://github.com/Lundalogik/lime-elements/runs/532907419?check_suite_focus=true) shouldn't have been green.)